test/FastTests/Client/Subscriptions/SubscriptionScripts.cs

```diff
@@ -772,6 +772,60 @@ await session.StoreAsync(new SupportCall()
         }
 
         [Fact]
+        public async Task CanHandleDates_Year()
+        {
+            using (var store = GetDocumentStore())
+            {
+                Server.ServerStore.Observer.Suspended = true;
```

Comment:
This is debug code and needs to be removed.
As the server is shared among tests, this can cause random test failures.